engine/src/main/java/org/terasology/rendering/nui/animation/SwipeMenuAnimationSystem.java

```diff
@@ -181,10 +181,14 @@ public void update(float delta) {
             // avoid skipping over fast animations on heavy load
             animDelta = 0.1f;
         }
+        if(animDelta == 0.0f) {
+            // in case time is paused
+            animDelta = 0.035f;
+        }
 
         flyIn.update(animDelta);
         flyOut.update(animDelta);
-    }
+    }   
```

Comment:
Trivial code style: extra spaces that serve no reason but to make the diff ever so slightly bigger